operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md

```diff
@@ -15,57 +15,94 @@ description: 50% Draft
-   2. Do not use screenshots rated at high quality
+   2. Do not use screenshots rated at low quality
       1. **\(Author owns\)** Rate each image High, Medium, Low quality
          1. **High** - Beautiful, highly compelling, grabs attention in a social stream 
          2. **Medium** - Tells a story clearly, readable text, right size 
          3. **Low** - Bare minimum to call an image 
    3. **\(Author owns\)** Draft Tweet text for all screenshots included
-      1. Contains version hashtag, e.g. \#mm518
+      1. Contains version hashtag, e.g. \#mm520
 
 ## **Style Guidelines**
 
 1. Highlight past successes: Cross link to promote traffic 
-2. Links tell a story: Links should indicate the content they lead to without clicking, avoid "read this blog post" and instead "learn more about the incident response alpha program" 
+2. Links tell a story: Links should indicate the content they lead to without clicking, avoid "read this blog post" and instead "learn more about the incident response alpha program"
 
 ## Screenshot Guidelines
 
 [Learn how to create screenshots and GIFs](https://handbook.mattermost.com/operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-screenshots-and-gifs)
 
+## Language Guidelines
+1. Specify Desktop app or Mobile App if a feature is not dependent on the server release
+2. Use common wording, e.g. say "Servers" instead of "Sites"; say `plugins` not `plug-ins`; say "Browser" instead of "Webapp"
+3. Avoid phrases that are too vague, such as "Better messaging experience"
+4. Do not use internal names of features, such as "Unread toasts", "Deep linking"
+
 ## One great screenshot a month
 
 1. Each monthly release R&D should aspire to come up with a "Hero" screenshot following guidelines to use as the splash image for the release.
 2. Communicate primary benefit in the title
-   * Use `plugins` not `plug-ins`
-3. If a screenshot isn't available to highlight in the release use a conceptual banner based on [R&D Twitter Banner Template](https://community.mattermost.com/files/6bc4cot5xbyi8cfxesgyx6aqdw/public?h=HKLm3FZvOvpylxVk7mUYmeaBQ_rFV_JCEAZ2w5z9fg4) \(shown below\)
+3. Keep the title short, use sentence case, and ensure there are no orphans or widows
+4. If a screenshot isn't available to highlight in the release use a conceptual banner based on [R&D Twitter Banner Template](https://community.mattermost.com/files/6bc4cot5xbyi8cfxesgyx6aqdw/public?h=HKLm3FZvOvpylxVk7mUYmeaBQ_rFV_JCEAZ2w5z9fg4) \(shown below\)
 
-![Screenshot for templates for announcements and Twitter banners, including guides](../../../.gitbook/assets/image%20%2862%29.png)
+[Example of tweet using banner template](https://twitter.com/iantien/status/1217457762772238336)
 
+![Screenshot for templates for announcements and Twitter banners, including guides](../../../.gitbook/assets/image%20%2862%29.png)
```

Comment:
Is this screenshot broken? No image renders in preview mode

Reply:
It looks fine on the handbook.

I haven't made changes to it and I'm not sure where the image comes from.